code/modules/admin/admin_verb_lists_vr.dm

```diff
@@ -393,6 +393,7 @@ var/list/admin_verbs_mod = list(
 )
 
 var/list/admin_verbs_event_manager = list(
+	/client/proc/toggle_vantag_hud,
```

Comment:
I would recommend adding this to admins as well since none of the Vorestation staff roles use the event role.